java/vector/src/test/java/org/apache/arrow/vector/TestValueVector.java

```diff
@@ -2639,6 +2641,61 @@ public void testEqualsWithIndexOutOfRange() {
     }
   }
 
+  @Test
+  public void testToString() {
+    try (final IntVector intVector = new IntVector("intVector", allocator);
+         final ListVector listVector = ListVector.empty("listVector", allocator);
+         final StructVector structVector = StructVector.empty("structVector", allocator)) {
+
```

Comment:
It would be good if we can evaluate empty range here.

Reply:
thanks, fixed.